__tests__/app.test.js

```diff
@@ -485,4 +485,64 @@ describe('DELETE request', () => {
       });
     
   });
-})
\ No newline at end of file
+})
+
+
+
+//GET Users
+
+describe("GET /api/users", () => {
+  test("GET 200 responds with users data in array", () => {
+    return request(app)
+      .get("/api/users")
+      .expect(200)
+      .then(({ body }) => {
+        const { user } = body;
+        expect(user).toBeInstanceOf(Array);
+      });
+  });
+
+  test(" return a length of users array", () => {
+    return request(app)
+      .get("/api/users")
+      .expect(200)
+      .then(({ body }) => {
+        const { user } = body;
+        expect(user.length).toBe(4);
+      });
+  });
+
+  test("Return objects should have all the properties", () => {
+    return request(app)
+      .get("/api/users")
+      .expect(200)
+      .then(({ body }) => {
+        const { user } = body;
+        user.forEach((user) => {
+          expect(user).toHaveProperty("username");
+          expect(user).toHaveProperty("name");
+          expect(user).toHaveProperty("avatar_url");
+        });
+      });
+  });
+
+  describe("Users error handler", () => {
+    test("GET:404 return appropriate message if there is no database", () => {
```

Comment:
This test isn't quite testing what the description says. This test isn't needed as you know there are currently users in the database - your other tests should account for there being any issues with the request